### PR TITLE
Egress Defend

### PR DESCRIPTION
**Pull request by EgressGlynn**
Required items, please complete
   
   Change(s):
   - Adding Egress Defend to Azure Sentinel

   Reason for Change(s):
   - To add Egress Defend to Azure Sentinel

   Version Updated:
   -  

   Testing Completed:
   - Yes

   Checked that the validations are passing and have addressed any issues that are present:
   -Yes

### CODE REVIEW

**Comment by EgressGlynn** [comment]
> @EgressGlynn please read the following Contributor License Agreement(CLA). If you agree with the CLA, please reply with the following information.
> 
> ```
> @microsoft-github-policy-service agree [company="{your company}"]
> ```
> 
> > Options:
> > 
> > * (default - no company specified) I have sole ownership of intellectual property rights to my Submissions and I am not making Submissions in the course of work for my employer.
> > 
> > ```
> > @microsoft-github-policy-service agree
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > 
> > * (when company given) I am making Submissions in the course of work for my employer (or my employer has intellectual property rights in my Submissions by contract or applicable law). I have permission from my employer to make Submissions and enter into this Agreement on behalf of my employer. By signing below, the defined term “You” includes me and my employer.
> > 
> > ```
> > @microsoft-github-policy-service agree company="Microsoft"
> > ```
> 
> Contributor License Agreement

@microsoft-github-policy-service agree company="Egress Software Technologies"

**Comment by v-prasadboke** [comment]
Hi @EgressGlynn thanks for committing the changes. I'll be examining this PR and you can expect an update till 01 Aug 2023.

File: Solutions/Egress Defend/Workbooks/WorkbooksMetadata.json
Comment (v-prasadboke) [inline Solutions/Egress Defend/Workbooks/WorkbooksMetadata.json:1]: Hello @EgressGlynn, Please remove this file. Workbook metadata should be added in this File https://github.com/Azure/Azure-Sentinel/blob/master/Tools/Create-Azure-Sentinel-Solution/V2/WorkbookMetadata/WorkbooksMetadata.json

**Comment by v-prasadboke** [comment]
Hello @EgressGlynn, Thank you for committing the changes. You can expect an update till 07 Aug 2023.

File: Sample Data/Egress Defend_RawLogs.json
Comment (v-prasadboke) [inline Sample Data/Egress Defend_RawLogs.json:1]: Hello @EgressGlynn, this sample data is not in proper format. Please refer to any of the sample data from SampleData folder of the repo.

File: Solutions/Egress Defend/Workbooks/WorkbooksMetadata.json
Comment (v-prasadboke) [inline Solutions/Egress Defend/Workbooks/WorkbooksMetadata.json:1]: And also Add this workbook metadata in this specified Location Tools/Create-Azure-Sentinel-Solution/V2/WorkbookMetadata/WorkbooksMetadata.json. 
And remove in from the solution folder.

**Comment by v-prasadboke** [comment]
Hello @EgressGlynn, Thanks for committing the changes. Will investigate this and update you before 10 Aug 2023.

**Comment by v-atulyadav** [comment]
Hi @EgressGlynn, 
We are still reviewing these contents and need some more time. We will provide you with a detailed review by 14 Aug 2023. Sorry for the inconvenience. Thanks

**Comment by v-prasadboke** [comment]
Hello @EgressGlynn, I committed some basic changes in the solution and all looks good. But just add a little more description regarding the solution in input file and in create ui as well. Thank you.

**Comment by v-prasadboke** [comment]
Hello @EgressGlynn, Thank you for committing the changes. Will have a look into it and update you by EOD. Thanks.

**Review by v-atulyadav — APPROVED** [review]

**Pull request merged**